libmproxy/flow.py

```diff
@@ -614,6 +619,43 @@ def killall(self, master):
         self.flows.kill_all(master)
 
 
+class Plugins(object):
```

Comment:
I think it's really superfluous for the content views - can we leave that out for now and just implement `ScriptContext.add_contentview()` to call `contentviews.add()` ? Let's keep the code in the history so that we can use it if we need to, but it seems like overkill for the contentviews.